tox.ini

```diff
@@ -115,7 +113,6 @@ passenv =
 skip_install = false
 usedevelop = true
 
-
```

Comment:
Let's keep double line separators between different kinds of sections (flake vs tox global vs tox testenvs). They signal the difference clearly.